projects/1.20-fabric/assets/sword-parry/frycparry/lang/zh_cn.json

```
@@ -0,0 +1,156 @@
+{
+  "key.category.frycparry.key_category_swordparry": "Sword Parry",
+  "key.frycparry.key_parry": "格挡（设为空即为右键格挡）",
+  "key.frycparry.key_dont_parry": "禁用格挡",

```

```suggestion
  "key.frycparry.key_parry": "招架（设为空即为右键招架）",
  "key.frycparry.key_dont_parry": "禁用招架",
```
请注意统一block和parry的翻译用词